packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.ts

```diff
@@ -58,7 +60,21 @@ export default class CardPayCreateMerchantWorkflowPrepaidCardChoiceComponent ext
     this.merchantRegistrationFee = yield this.layer2Network.strategy.merchantRegistrationFee();
   }
 
+  get prepaidCards() {
+    return this.layer2Network.safes.value.filterBy(
+      'type',
+      'prepaid-card'
+    ) as PrepaidCardSafe[];
+  }
+
+  @action choosePrepaidCard(card: PrepaidCardSafe) {
+    this.selectedPrepaidCard = card;
+  }
+
   @action createMerchant() {
+    if (this.isCtaDisabled) {
+      return;
+    }
```

Comment:
They shouldn’t be able to trigger this action if the button is disabled, is there another reason to have this guard? 😯

Reply:
> They shouldn’t be able to trigger this action if the button is disabled, is there another reason to have this guard? 😯

Typescript gives an error